### ISSUE TITLE
[Machine Learning] How to build a Text Classification model Using BERT and Tensorflow

### ISSUE THREAD

**Issue by Bravinsimiyu** [section-engineering-education/engineering-education#5225]
#### Proposed title of article
[Machine Learning] How to build a Text Classification model Using BERT and Tensorflow

#### Proposed article introduction
Text classification is a subset of machine learning that involves classifying text into predefined categories. Text classification is a fundamental task in natural language processing. Some of the examples of text classification are sentiment analysis, topic labeling, spam detection, and intent detection.

In this tutorial, we will build a spam detection model.  The spam detection model will classify emails as spam or not spam. This is used to detect unsolicited and unwanted emails. We will build this model using BERT and Tensorflow. BERT will be used to generate sentence encoding for all emails and after that, we will use a simple neural network with one drop out layer and one output layer. It's developed by Google.
Finally, we will use Tensorflow to build the neural networks. Tensorflow will build the input and output layers of our machine learning model.

#### Key takeaways 
1. What is text classification and various examples of text classification.
2. What is Bert Model.
3. How to install a Bert model.
4. Exploring Tenserflow functions.
5. Building a spam filtering model Using BERT and Tensorflow.

#### Article quality
This article is unique since we will be building a unique model. The model will be a spam filtering model which is a type of text classification. We will also use Bert to generate sentence encoding for all emails and after that, we will use a simple neural network with one drop out layer and one output layer. We will also implement dropout regularization to handle overfitting during model building.

#### References
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by ahmadmardeni1** [comment]
Sounds like a helpful topic - let's please be sure it adds value beyond what is in any official docs and/or what is covered in other blog sites. (the articles should go beyond a basic explanation - and it is always best to reference any EngEd article and build upon it). @Bravinsimiyu 

Please be attentive to grammar/readability and make sure that you put your article through a thorough editing review prior to submitting it for final approval. (There are some great free tools that we reference in EngEd resources.)
ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.

Please be sure to double-check that it does not overlap with any existing EngEd articles, articles on other blog sites, or any incoming EngEd topic suggestions (if you haven't already) to avoid any potential article closure, please reference any relevant EngEd articles in yours. - Approved
